### PR TITLE
Add package `imake` v1.0.8

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add package version 1.0.8

###### Change Log  

- imake: add package version 1.0.8
- imake: appease linter
- imake: cgmanifest: add entry
- imake: license: add entry

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Local Build with RUN_CHECK=y

### CODE REVIEW

File: SPECS-EXTENDED/imake/imake.spec [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+%{_bindir}/mkdirhier
+%{_bindir}/mkhtmlindex
+%{_bindir}/revpath
+%{_bindir}/xmkmf
+%dir %{_datadir}/X11/config
+%{_datadir}/X11/config/*.cf
+%{_datadir}/X11/config/*.def
+%{_datadir}/X11/config/*.rules
+%{_datadir}/X11/config/*.tmpl
+#%%dir %%{_mandir}/man1x
```
Comment (PawelWMS) [inline SPECS-EXTENDED/imake/imake.spec:100]:
```suggestion
```

File: SPECS-EXTENDED/imake/imake.spec [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+# Install everything
+{
+   for pkg in imake makedepend gccmakedep lndir xorg-cf-files ; do
+      pushd $pkg-*
+      make install DESTDIR=%{buildroot}
+      popd
+   done
+}
+
+%files
```
Comment (PawelWMS) [inline SPECS-EXTENDED/imake/imake.spec:82]: Missing `%license` macro.

File: SPECS-EXTENDED/imake/imake.spec [hunk 3]
```diff
@@ -0,0 +1,200 @@
+S4280594522ummary:        imake source code configuration and build system
```
Comment (neha170) [inline SPECS-EXTENDED/imake/imake.spec:1]: nit: typo in Summary

File: SPECS-EXTENDED/imake/imake.spec [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+Name:           imake
+Version:        1.0.8
+Release:        4%{?dist}
```
Comment (neha170) [inline SPECS-EXTENDED/imake/imake.spec:4]: Release should be 5

**Review by neha170 — APPROVED** [review]
Please address 2 small edits, looks good otherwise

**Review by PawelWMS — APPROVED** [review]
+1 to Neha's comments, otherwise looks good to me as well.

File: SPECS-EXTENDED/imake/imake.spec (hunk 3 above)
Comment (mfrw) [reply SPECS-EXTENDED/imake/imake.spec:1]: OMG :( thanks

**Pull request merged**